<ui version="4.0" >
 <class>pqLinksEditor</class>
 <widget class="QDialog" name="pqLinksEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QTreeView" name="ObjectTreeProxy2" />
       </item>
       <item row="0" column="0" >
        <widget class="QTreeView" name="ObjectTreeProxy1" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QListWidget" name="Property1List" />
       </item>
       <item row="0" column="0" >
        <widget class="QTreeView" name="ObjectTreeProperty1" />
       </item>
       <item row="1" column="1" >
        <widget class="QListWidget" name="Property2List" />
       </item>
       <item row="0" column="1" >
        <widget class="QTreeView" name="ObjectTreeProperty2" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit" />
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" >
       <item>
        <property name="text" >
         <string>Object Link</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Property Link</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqLinksEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqLinksEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>481</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
